src/api/app/models/concerns/workflow_run_payload.rb

```diff
@@ -68,67 +60,67 @@ def ignored_push_event?
   private
 
   def github_push_event?
-    @payload[:scm] == 'github' && @payload[:event] == 'push' && @payload.fetch(:ref, '').start_with?('refs/heads/')
+    scm_vendor == 'github' && payload[:event] == 'push' && payload.fetch(:ref, '').start_with?('refs/heads/')
```

Comment:
The first part of this condition is kind of given now right? I mean the method is called `github....`